mapedit/studio.h

```diff
@@ -98,7 +98,6 @@ using Msg_callback = void (*)(Exult_server::Msg_type id,
 #define WIN32_LEAN_AND_MEAN
 #endif
 #include <windows.h>
-#include "windrag.h"
 #define C_EXPORT extern "C" __declspec(dllexport)
 #endif
```

Comment:
Is any of this still needed?

Reply:
(and this shuold be from lines 94-102, but gh would not let me select above line 98)